charts/gateway/production-values.yaml

```diff
@@ -9,6 +9,8 @@ license:
   accept: false
   # existingSecretName: ssg-license
 
+disklessConfig: true
```

Comment:
Need to update with new schema. See values.yaml

Reply:
Addressed in https://github.com/CAAPIM/apim-charts/pull/318/commits/9b9d8d060847d479d6993d9698df8e6afd3cda32